### ISSUE TITLE
RocketChip: Project Formation

### ISSUE THREAD

**Issue by bensternthal** [chipsalliance/tac#169]
## Project Documentation & Links

Project Name: RocketChip
Project Repository: https://github.com/chipsalliance/rocket-chip
Project Data File/PR: https://github.com/chipsalliance/tac/blob/main/projects/project-data-files/rocket-chip.yml

### Project Checklist

- [x] TAC Vote **Grandfathered**
- [x] New Project Created In LFX - Exists as Rocket WG - should fix
- [x] [Legal Submission](https://drive.google.com/drive/folders/1XGFtJeIrgSQTZVbcpuVlLV39q0Xu9ts9)
   - [x] Project formation form submitted by project 
   - [x] Technical Charter Draft Approved
   - [ ] Contribution Agreement Approved
   - [ ] Trademark Docs Approved
- [ ] Board Vote  **Date**
- [ ] Project made active in PCC by notifying the formation team
- [ ] Merge PR & Confirm Website Is Accurate
- [ ] Add ‘thelinuxfoundation’ as an owner of the GitHub org
- [ ] Transfer any domains
- [ ] Onboard to LFX/Community Metrics/Insights
- [ ] Enforce CLA on repo(s)
- [ ] Accept/adopt LF code of conduct
- [ ] Enable security vulnerability reporting process

**Comment by bensternthal** [comment]
Emailed project contact.

**Comment by bensternthal** [comment]
Contact confirmed, requested intake form be filled out.

**Comment by bensternthal** [comment]
Form filled out, blocked waiting on project formation steps being completed. Have not heard back from stakeholder.

**Comment by bensternthal** [comment]
Reached out to Jiuyang L again to check on next steps.

**Comment by bensternthal** [comment]
Target completion next week.

**Comment by bensternthal** [comment]
Checked in via email Aug 15

**Comment by bensternthal** [comment]
Jiuyang approval of legal docs Aug 20.

**Comment by bensternthal** [comment]
Scott requested "Thank you both. Ben, can you have the details completed for the [contribution agreement](https://docs.google.com/document/d/1rX5rYWr5UM2Xe45TVeyGge0YdLoCRKc5B8zgPncDFYQ/edit) and work with Todd to have that docusign sent out? Thanks,"

I will complete post PTO

**Comment by bensternthal** [comment]
https://github.com/chipsalliance/rocket-chip/pull/3683

**Comment by bensternthal** [comment]
Sent trademark email, will need to decide if we want to keep or transfer.

**Comment by bensternthal** [comment]
Provided two week deadline to make a decision, or we will need to start offboading the project.

**Comment by bensternthal** [comment]
Jiuyang is working to get a meeting with Krste.

**Comment by bensternthal** [comment]
Reached out for update on name again.

**Comment by bensternthal** [comment]
Meeting scheduled for next monday to discuss.

**Comment by bensternthal** [comment]
On hold pending contribution agreement assignment of common law trademark.
